tests/integration_tests/pretrained_models_test.py

```diff
@@ -189,7 +189,29 @@ def setUp(self) -> None:
                     "loss_logging_items_names": ['smooth_l1', 'closs', 'Loss'],
                     "metric_to_watch": "mAP@0.50:0.95",
                     "greater_metric_to_watch_is_better": True
-                }
+                },
+            "yolox":
+                {"max_epochs": 3,
+                 "lr_mode": "cosine",
+                 "cosine_final_lr_ratio": 0.05,
+                 "warmup_bias_lr": 0.0,
+                 "warmup_momentum": 0.9,
+                 "initial_lr": 0.02,
+                 "loss": "yolox_loss",
+                 "criterion_params": {
+                     "strides": [8, 16, 32],  # output strides of all yolo outputs
+                     "num_classes": len(self.transfer_detection_dataset['ssd_lite_mobilenet_v2'].classes)},
+
+                 "loss_logging_items_names": ["iou", "obj", "cls", "l1", "num_fg", "Loss"],
+
+                 "train_metrics_list": [],
+                 "valid_metrics_list": [
+                     DetectionMetrics(
+                         post_prediction_callback=SSDPostPredictCallback(),
```

Comment:
Could you please explain what is added here? Is it a mistake that SSD things are used for the DetectionMetrics under yolox key or is it supposed to be like this?

Reply:
Post prediction callback is a mistake, but worked since they actually do the exact same thing (:
I will to the backlog that we should drop it and just rename the yolo one.

Regarding the other stuff- I'm just using the transfer learning dataset, but since its confusing I changed it to just be the same dataset (I.e without the dictionary and all the transfer learning detection tests just use the same one)